docs/en/ingest-management/elastic-agent/scaling-on-kubernetes.asciidoc

```diff
@@ -0,0 +1,220 @@
+[[scaling-on-kubernetes]]
+= Scaling {agent} on kubernetes
+
+- Please follow the steps to run the {agent} on <<running-on-kubernetes-managed-by-fleet>> page.
```

Comment:
Can we be more verbose here with what we mean? Sth like "In order to first deploy Elastic Agent on k8s follow the steps at X....."

Reply:
Check latest commit and let me know if u agree